main/ertp/ERTP-Guide/AmountMath.md

```diff
@@ -0,0 +1,197 @@
+# Amount Math
+
+![AmountMath methods](./assets/amountMath.svg) 
+
+`issuer` objects must be able to deposit and withdraw assets from a
+`purse` and manipulate `payment` amounts. This 
+requires being able to add and subtract digital assets. To do this,
+and other operations on an `amount`, an `issuer` uses `amountMath`methods. 
+
+In addition to math operations, `amountMath` methods check their
+arguments' brands, throwing an error if the wrong `brand` was used.
+
+There are three different kinds of `amountMath` methods, each of which
+implements the same methods. When you create an `issuer`, you
+do so for a specific `brand`. You thus have to specify which kind of
+`amountMath` that `brand` uses. The 
+correct kind is automatically used whenever an `amountMath` method
+is used on an `amount` with that `brand`. The kinds are: 
+- `nat`: Used with fungible assets (the values are natural numbers).
+- `strSet`: Used with non-fungible assets (the values are strings).
+- `set`: Used with sets of objects, primarily non-fungible assets.
+
+`makeIssuerKit(allegedName, amountMathKind)` creates a new `issuer`.
+The second, optional, argument specifies which kind
+of `amountMath` your contract uses for the `brand` in a one-to-one
+association with the new `issuer`. It defaults to `nat` if not given. For example: 
+```js
+makeIssuerKit('quatloos`); // Defaults to 'nat'
+makeIssuerKit('quatloos', 'strSet');
+makeIssuerKit('quatloos, 'set');
+```
+`AmountMath` has:
+- Three information getting methods (get brand, get value, and get
+`amountMath` kind)
+- Four comparison methods (is an amount empty, is an amount valid,
+are two amounts equal, and is one amount greater than or equal to another)
+- Two manipulator methods (add and subtract amounts)
+- Two amount creation method (create an `amount` with the
+  `amountMath`'s associated `brand`, and get a new, empty, `amount`)
+
+The following is a brief description and example of each `amountMath` method. For
+more detail, click the method's name to go to its entry in the [ERTP
+API Reference](https://agoric.com/documentation/ertp/api/#ertp-api).
```

Comment:
We should compare to my re-writes after I finish hiding amountMathHelpers.

Reply:
Yep. I wrote under the assumption that things would stay the same, save for moving the kinds up to amountMath but will confirm.